tools/Makefile

```diff
@@ -312,6 +312,20 @@ clean::
 clean::
 	rm -f "objinfo_helper$(EXE)" "objinfo_helper$(EXE).manifest"
 
+# Eventlog metadata file
+
+eventlog_metadata: $(EVENTLOG_METADATA)
+	cat eventlog_metadata.in | sed "s/@endianness/$(EVENTLOG_ENDIANNESS)/g" > eventlog_metadata
```

Comment:
Remove unnecessary use of `cat`. You can use `sed "s/@endianness/$(EVENTLOG_ENDIANNESS)/g" eventlog_metadata.in > eventlog_metadata` instead?